pkgs/applications/graphics/hdrmerge/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, extra-cmake-modules
+, qtbase
+, wrapQtAppsHook
+, libraw
+, exiv2
+, zlib
+, alglib
+, pkgconfig
+, makeDesktopItem
+}:
+
+stdenv.mkDerivation rec {
+  name = "hdrmerge";
+
+  src = fetchFromGitHub {
+    owner = "jcelaya";
+    repo = "hdrmerge";
+    rev = "master";
+    sha256 = "1bzf9wawbdvdbv57hnrmh0gpjfi5hamgf2nwh2yzd4sh1ssfa8jz";
+  };
+
+  # dev dependencies
+  nativeBuildInputs = [
+    cmake
+    pkgconfig
+    wrapQtAppsHook
+  ];
+
+  # runtime dependencies
+  buildInputs = [ qtbase libraw exiv2 zlib alglib ];
+
+  cmakeFlags = [
+    "-DALGLIB_DIR:PATH=${alglib}"
+  ];
+
+  patches = [
+    # patch FindAlglib.cmake to respect ALGLIB_DIR
+    # see https://github.com/jcelaya/hdrmerge/pull/213
+    ./patch-hdrmerge-CMake.patch
+  ];
+
+  desktopItem = makeDesktopItem {
+    name = "HDRMerge";
+    genericName = "HDR raw image merge";
+    desktopName = "HDRMerge";
+    comment = meta.description;
+    icon = "hdrmerge";
+    exec = "@out@/bin/hdrmerge -F";
+    categories = "Graphics;";
+    mimeType = "image/x-dcraw;image/x-adobe-dng";
+    terminal = false;
+  };
+
+  postInstallPhase =
+    ''
```

Comment:
```suggestion
  postInstall = ''
```

Reply:
When I apply this, I get `cp: cannot stat 'data/images/logo.png': No such file or directory` which is weird. What is the difference between `postInstallPhase` and `postInstall`?

Reply:
`stdenv` doesn't execute `postInstallPhase` hook so you actually not copying those files. `cmake` hook puts script in a subdirectory. I suggested the fix.